examples/teleportation.ipynb

```diff
@@ -0,0 +1,634 @@
+{
```

Comment:
<p>Is there an option to only return the target Hilbert space (or qubits) and trace out the rest qubits? It might look nicer in the notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/qutip/qutip-notebooks/pull/108/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
I am not quite sure I follow, what do you mean by returning the qubits here?

Reply:
As in, do we want to reduce the dimension of the hilbert space while doing measurement ?

Reply:
Oh sorry, I was in the `ReviewNB` mode, if you click the purple button `ReviewNB` above, you can see which notebook cell I was referring to. I just realize that you see nothing on GitHub

To your question, no, in the circuit, it's good to keep the size of the Hilbert space. I was just wondering if there are ways to show the result without huge matrices. They are going to take a big space and block the text flow. This is just a minor thing and can be dealt with when you want to finalize the notebook.

Reply:
Maybe we can write the output in terms of bra-ket notation?